src/localzigzag.jl

```diff
@@ -31,7 +31,9 @@ function λ(G, ∇ϕ, i, x, θ, Z::LocalZigZag)
 end
 
 #ab(G, i, x, θ, c, Z::LocalZigZag) = Z.Γ[:,i]'*(x-Z.μ)*θ[i] + c[i]*sqrt(sum(abs2(x[j]-Z.μ[j]) for j in neighbours(G, i))), 1.0
-ab(G, i, x, θ, c, Z::LocalZigZag) = c[i]*sqrt(sum(abs2(x[j]-Z.μ[j]) for j in neighbours(G, i))), 1.0
+#ab(G, i, x, θ, c, Z::LocalZigZag) = c[i]*sqrt(sum(abs2(x[j]-Z.μ[j]) for j in neighbours(G, i))), 1.0
+#ab(G, i, x, θ, c, Z::LocalZigZag) = c[i] + mean(θ[j]*(x[j] - Z.μ[j]) for j in neighbours(G, i)), 1.0
+ab(G, i, x, θ, c, Z::LocalZigZag) = c[i] + θ[i]*sum(Z.Γ[i, :][j]*(x[j] - Z.μ[j]) for j in neighbours(G, i)), θ[i]*sum(Z.Γ[i, :][j]*θ[j] for j in neighbours(G, i))
```

Comment:
This looks like it would be working! The `c` accounts for the mismatch between `Z.Γ` and the true potential.